.idea/codeStyles/Project.xml

```diff
@@ -6,15 +6,6 @@
           <package name="kotlinx.android.synthetic" alias="false" withSubpackages="true" />
         </value>
       </option>
-      <option name="PACKAGES_IMPORT_LAYOUT">
```

Comment:
Is this coming from the IntelliJ update?

Reply:
Yup...I just switched to AS